planner/core/cache.go

```diff
@@ -206,21 +205,16 @@ type PlanCacheValue struct {
 	Plan              Plan
 	OutPutNames       []*types.FieldName
 	TblInfo2UnionScan map[*model.TableInfo]bool
-	TxtVarTypes       FieldSlice // variable types under text protocol
-	BinVarTypes       []byte     // variable types under binary protocol
-	IsBinProto        bool       // whether this plan is under binary protocol
+	TxtVarTypes       FieldSlice
```

Comment:
I'll rename `TxtVars`, `Args`, `Parameters`, `Vals` to the unified keyword `Params` in the next PR.